File: benchmark/code/factorials/factorials.py
#!/usr/bin/python

def factorial(n):
	f = 1
	for i in range(1, n):
		f *= i
	return f

c = 0
for i in range(0, 30):
	for j in range(1, 1000):
		c += factorial(j) + i
	c = c % 1000000

print(c)

File: benchmark/code/factorials/factorials.java
public class factorials {

	static long factorial(int number) {
		long n = 1;
		for (int i = 1; i < number; i++) {
			n = n * i;
		}
		return n;
	}

	public static void main(String[] args) {
		long c = 0;
		for (int i = 1; i < 3000; ++i) {
			for (int j = 1; j < 1000; ++j) {
				c += factorial(j) + i;
			}
			c = c % 1000000;
		}
		System.out.println(c);
	}
}
